VueJS/index.js

```diff
@@ -1,13 +1,84 @@
-import List from './components/List.js'
-import ListItem from './components/ListItem.js'
+Vue.component('todo-list', {
+	template: `<div>
+		<ul>
+			<todo-item
+				v-for="(todo, index) in todos"
+				v-bind:todo="todo"
+				v-on:save="$emit('save', {index: index, message: $event})"
+				v-on:remove="$emit('remove', index)"
+			/>\
+		</ul>
+		<button v-on:click="$emit('add')">Add new todo</button>
+		</div>`,
+	props: ['todos']
+})
+
+Vue.component('todo-item', {
+  template: `<li>
+	<template v-if="editing">
+		{{ todo.id }} {{ todo.message }}
```

Comment:
```suggestion
		{{ todo.message }}
```
The `id` is for debugging right?